From 99ea614bff3d383fbd6910ae0d66912fc15eba57 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Fri, 30 Aug 2013 08:51:23 +0200
Subject: [PATCH 04/29] Disable usb-uas

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <95623a002492b1f8117e0f99e2b1163f2a2a4071.1377851454.git.mrezanin@redhat.com>
Patchwork-id: 53934
O-Subject: [RHEL7 qemu-kvm PATCHv2 3/6] Disable usb-uas
Bugzilla: 903914
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Michal Novotny <minovotn@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

BZ: 903914

As usb3 streams are still broken, we do not need usb-uas device.

v2:
  - new patch
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/usb.mak | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/usb.mak |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/default-configs/usb.mak b/default-configs/usb.mak
index 39a3552..55c37f0 100644
--- a/default-configs/usb.mak
+++ b/default-configs/usb.mak
@@ -1,3 +1,2 @@
 CONFIG_USB_STORAGE_BOT=y
-CONFIG_USB_STORAGE_UAS=y
 CONFIG_USB_SMARTCARD=y
-- 
1.7.1

